### PR TITLE
Improve and fix various things

### PR DESCRIPTION
**Pull request by shizonic**

### CODE REVIEW

File: labs/02_cli.md [hunk 1]
```diff
@@ -4,17 +4,39 @@ In diesem Lab werden wir gemeinsam das CLI-Tool `odo` installieren und konfiguri
...
+
+Dieser Schritt ist optional und funktioniert nicht auf Windows. Damit Command Completion auf macOS funktioniert, muss bspw. via `brew` das Paket `bash-completion` installiert werden.
+
+`oc` bietet eine Command Completion, die gem. [Dokumention](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/configuring-cli.html#cli-enabling-tab-completion_cli-configuring-cli) eingerichtet werden kann.
+
+__Tipp__:
+Alternativ kann die Bash Command Completion auch mittels folgenden Befehlen im Terminal installiert werden:
+
+```
+sudo oc completion bash > /etc/bash_completion.d/oc_bash_completion
```
Comment (bliemli) [inline labs/02_cli.md]: This does not work as there's no sudo rights after a redirect.

```suggestion
sudo sh -c "oc completion bash > /etc/bash_completion.d/oc"
```

File: labs/02_cli.md [hunk 2]
```diff
@@ -4,17 +4,39 @@ In diesem Lab werden wir gemeinsam das CLI-Tool `odo` installieren und konfiguri
...
 Via `odo` kommunizieren wir mit OpenShift. Es wurde mit OpenShift 4 neu ins Leben gerufen und fokussiert sich auf die Entwickler-relevanten Aufgaben.
 
-Der __oc client__ stellt ein Interface zu OpenShift bereit.
-
 `odo` ist in [Go](https://github.com/openshift/odo) programmiert und kommt als einzelnes Binary für die gängigsten Betriebsysteme daher.
 
-## Installation
+### Installation
 
 Folgen Sie für die Installation der [offiziellen Installationsdokumentation](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_developer_cli/installing-odo.html).
```
Comment (bliemli) [inline labs/02_cli.md]: (It seems like I can't comment on more than 10 lines, this comment applies to lines 33 to 51 however).

`odo` is not yet used during the labs, so there's no point in installing it.

File: labs/06_scale.md [hunk 3]
```diff
@@ -98,13 +98,13 @@ Wie können Sie die Anzahl Replicas via Web Console steuern?
...
-Ersetzen Sie `[HOSTNAME]` mit dem Hostname Ihrer definierten Route:
-
 __Tipp__:
+Um den entsprechenden Hostname anzuzeigen, kann folgender Befehl verwendet werden:
+
 `oc get route -o custom-columns=NAME:.metadata.name,HOSTNAME:.spec.host`
 
 ```bash
-while true; do sleep 1; ( { curl -fs http://[HOSTNAME]/health/; date "+ TIME: %H:%M:%S,%3N" ;} & ) 2>/dev/null; done
+export HNAME=$(oc get route -o custom-columns=:.spec.host | tail -n 1); while true; do sleep 1; ( { curl -fs http://${HNAME}/health/; date "+ TIME: %H:%M:%S,%3N" ;} & ) 2>/dev/null; done
```
Comment (bliemli) [inline labs/06_scale.md]: This command is too abstract in order for participants to easily understand what it does. Also, as soon as there's two routes inside the project (which already happened several times) it's not guaranteed that the correct route is taken.

**Review by bliemli — CHANGES_REQUESTED** [review]
Thank you for your great review!

File: labs/02_cli.md (hunk 1 above)
Comment (shizonic) [reply labs/02_cli.md]: I see. `oc completion bash | sudo tee /etc/bash_completion.d/oc_bash_completion` might work too.

File: labs/06_scale.md (hunk 3 above)
Comment (shizonic) [reply labs/06_scale.md]: I'll revert it.

**Review by bliemli — APPROVED** [review]

**Pull request merged**